The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/11617954474/job/32354312535) [(plain)](https://github.com/rust-lang/rust/commit/57620314c2ae13c1dd18a15666e9b47e0766f94f/checks/32354312535/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain

COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/

# NOTE: intentionally uses python2 for x.py so we can test it still works.
# validate-toolstate only runs in our CI, so it's ok for it to only support python3.
ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test \
           --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile --allow-unsafe --generate-hashes reuse-requirements.in
---
#13 2.776 Building wheels for collected packages: reuse
#13 2.777   Building wheel for reuse (pyproject.toml): started
#13 3.031   Building wheel for reuse (pyproject.toml): finished with status 'done'
#13 3.032   Created wheel for reuse: filename=reuse-4.0.3-cp310-cp310-manylinux_2_35_x86_64.whl size=132720 sha256=026f3bb0f1aa8090b861fd0a0939cb1a782396d84c8aab7875096557d637a0f6
#13 3.032   Stored in directory: /tmp/pip-ephem-wheel-cache-tv2dgkmp/wheels/3d/8d/0a/e0fc6aba4494b28a967ab5eaf951c121d9c677958714e34532
#13 3.035 Installing collected packages: boolean-py, binaryornot, tomlkit, reuse, python-debian, markupsafe, license-expression, jinja2, chardet, attrs
#13 3.437 Successfully installed attrs-23.2.0 binaryornot-0.4.4 boolean-py-4.0 chardet-5.2.0 jinja2-3.1.4 license-expression-30.3.0 markupsafe-2.1.5 python-debian-0.1.49 reuse-4.0.3 tomlkit-0.13.0
#13 3.438 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#13 3.973 Collecting virtualenv
#13 3.973 Collecting virtualenv
#13 4.010   Downloading virtualenv-20.27.1-py3-none-any.whl (3.1 MB)
#13 4.068      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.1/3.1 MB 56.7 MB/s eta 0:00:00
#13 4.124 Collecting filelock<4,>=3.12.2
#13 4.128   Downloading filelock-3.16.1-py3-none-any.whl (16 kB)
#13 4.161 Collecting platformdirs<5,>=3.9.1
#13 4.165   Downloading platformdirs-4.3.6-py3-none-any.whl (18 kB)
#13 4.183 Collecting distlib<1,>=0.3.7
#13 4.201      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 469.0/469.0 KB 38.5 MB/s eta 0:00:00
#13 4.201      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 469.0/469.0 KB 38.5 MB/s eta 0:00:00
#13 4.282 Installing collected packages: distlib, platformdirs, filelock, virtualenv
#13 4.459 Successfully installed distlib-0.3.9 filelock-3.16.1 platformdirs-4.3.6 virtualenv-20.27.1
#13 DONE 4.5s

#14 [7/8] COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
#14 DONE 0.0s
---
DirectMap4k:      231360 kB
DirectMap2M:     7108608 kB
DirectMap1G:    11534336 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
    Finished `dev` profile [unoptimized] target(s) in 0.04s
##[endgroup]
downloading https://static.rust-lang.org/dist/2024-10-16/rustfmt-nightly-x86_64-unknown-linux-gnu.tar.xz
extracting /checkout/obj/build/cache/2024-10-16/rustfmt-nightly-x86_64-unknown-linux-gnu.tar.xz to /checkout/obj/build/x86_64-unknown-linux-gnu/rustfmt
---
fmt check
fmt: checked 5645 files
tidy check
tidy: Skipping binary file check, read-only filesystem
##[error]tidy error: /checkout/compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp:833: line longer than 100 chars
removing old virtual environment
creating virtual environment at '/checkout/obj/build/venv' using 'python3.10'
Requirement already satisfied: pip in ./build/venv/lib/python3.10/site-packages (24.3.1)
All checks passed!
checking C++ file formatting
checking C++ file formatting
/checkout/compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp:830:42: error: code should be clang-formatted [-Wclang-format-violations]
          MPM.addPass(LowerTypeTestsPass(/*ExportSummary=*/nullptr,
                                         ^
/checkout/compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp:830:68: error: code should be clang-formatted [-Wclang-format-violations]
          MPM.addPass(LowerTypeTestsPass(/*ExportSummary=*/nullptr,
                                                                   ^
/checkout/compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp:832:27: error: code should be clang-formatted [-Wclang-format-violations]
#if LLVM_VERSION_GE(20, 0)
                          ^
/checkout/compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp:834:6: error: code should be clang-formatted [-Wclang-format-violations]
#else


clang-format linting failed! Printing diff suggestions:
--- /checkout/compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp (actual)
+++ /checkout/compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp (formatted)
@@ -826,14 +826,15 @@
   if (!IsLinkerPluginLTO && SanitizerOptions && SanitizerOptions->SanitizeCFI &&
       !NoPrepopulatePasses) {
     PipelineStartEPCallbacks.push_back(
         [](ModulePassManager &MPM, OptimizationLevel Level) {
-          MPM.addPass(LowerTypeTestsPass(/*ExportSummary=*/nullptr,
-                                         /*ImportSummary=*/nullptr,
+          MPM.addPass(LowerTypeTestsPass(
+              /*ExportSummary=*/nullptr,
+              /*ImportSummary=*/nullptr,
 #if LLVM_VERSION_GE(20, 0)
-                                         /*DropTypeTests=*/llvm::lowertypetests::DropTestKind::None));
+              /*DropTypeTests=*/llvm::lowertypetests::DropTestKind::None));
 #else
-                                         /*DropTypeTests=*/false));
+              /*DropTypeTests=*/false));
         });
   }
 


some tidy checks failed
tidy error: checks with external tool 'clang-format' failed
Command has failed. Rerun with -v to see more details.
  local time: Thu Oct 31 18:51:03 UTC 2024
  network time: Thu, 31 Oct 2024 18:51:04 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp
+++ b/compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp
@@ -827,9 +827,9 @@ extern "C" LLVMRustResult LLVMRustOptimize(
       !NoPrepopulatePasses) {
     PipelineStartEPCallbacks.push_back(
         [](ModulePassManager &MPM, OptimizationLevel Level) {
-          MPM.addPass(LowerTypeTestsPass(/*ExportSummary=*/nullptr,
-                                         /*ImportSummary=*/nullptr,
-                                         /*DropTypeTests=*/false));
+          MPM.addPass(LowerTypeTestsPass(
+              /*ExportSummary=*/nullptr,
+              /*ImportSummary=*/nullptr));
         });
   }
 

